pkgs/by-name/ra/radiance/package.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, cmake
+, libGLU
+, fetchFromGitHub
```

I'm terribly sorry, @RobWalt, I wasn't aware that there has been a change in preference in regards to where the comma should be placed. ☹️ Running the formula through `nix run github:NixOS/nixfmt` gives the following changes:
```suggestion
{
  lib,
  stdenv,
  cmake,
  libGLU,
  fetchFromGitHub,
```

---

No worries at all! Thanks for paying attention to all of the details. I'd rather have someone question all the tiny things instead of having no conversions here at all ❤️

---

That's great to hear, thank you for appreciating it 🙂